imagemanager/src/main/java/com/io/image/manager/controller/ConnectivityController.java

```diff
@@ -18,7 +18,15 @@ public ConnectivityController(ConnectivityService connectivityService){
             method = RequestMethod.GET,
             produces = MediaType.APPLICATION_JSON_VALUE
     )
-    public String connectivityCheck(){
-        return String.valueOf(connectivityService.isConnectivityEstablished());
+    public ConnectivityCheckResponse connectivityCheck(){
+        return new ConnectivityCheckResponse(connectivityService.isConnectivityEstablished());
+    }
+
+    public static class ConnectivityCheckResponse {
+        public boolean status;
```

Comment:
status name for a boolean variable is not descriptive enough, `isOnline` is probably better